src/components/ui/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,42 @@
+import { forwardRef } from 'react'
+import type { HTMLAttributes } from 'react'
+import Icon from 'src/components/ui/Icon'
+
+export interface ToggleProps
+  extends HTMLAttributes<Omit<HTMLInputElement, 'disabled'>> {
+  label?: string
+  disabled?: boolean
+  variant?: 'horizontal' | 'vertical'
+}
+
+const Toggle = forwardRef<HTMLInputElement, ToggleProps>(function Toggle(
+  { label, disabled, variant = 'horizontal', ...otherProps }: ToggleProps,
+  ref
+) {
+  return (
+    <div data-fs-toggle data-fs-toggle-variant={variant}>
+      <div data-fs-toggle-container>
+        <input
+          ref={ref}
+          id="toggle"
+          name="toggle"
```

Comment:
I'm not sure these attributes should be hardcoded. If someone uses more than one of them on the same page they will have problems toggling them separately.

Reply:
You're right, I followed the same we used for `InputText` component